Transport/SendinBlueApiTransport.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace Drixs6o9\SendinblueMailerBundle\Transport;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Mailer\Envelope;
+use Symfony\Component\Mailer\Exception\HttpTransportException;
+use Symfony\Component\Mailer\SentMessage;
+use Symfony\Component\Mailer\Transport\AbstractApiTransport;
+use Symfony\Component\Mime\Address;
+use Symfony\Component\Mime\Email;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Contracts\HttpClient\ResponseInterface;
+
+class SendinblueApiTransport extends AbstractApiTransport
```

Comment:
You've forgot to rename the file :)

Reply:
You asked for this change: 
![image](https://user-images.githubusercontent.com/4287777/84058702-dec94e00-a9b9-11ea-8370-a84b3d996849.png)

Reply:
Sorry, it's a habit to follow PSR-4 and it seemed obvious to me.
https://www.php-fig.org/psr/psr-4/

Reply:
Should I keep like this or revert?